traveler-ios-ui/TravelerKitUI/Storyboards/Parking.storyboard

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In this storyboard there's an empty view controler that says "location services not available". That looks to me like an error view controller. It would be a good idea to use a storyboard reference to that view controller and modify the text accordingly. This is to have consistency across our error screens.